src/core/aio.c

```diff
@@ -505,34 +508,57 @@ nni_aio_list_active(nni_aio *aio)
 static void
 nni_aio_expire_add(nni_aio *aio)
 {
-	nni_list *list = &aio->a_expire_q->eq_list;
-	nni_aio * prev;
+	nni_aio_expire_q * eq = aio->a_expire_q;
+	if (eq->eq_len > eq->eq_cap) {
+		nni_aio ** new_list =
```

Comment:
very minor code formatting nit.  Please run this code through clang format.

We use the unary pointer operator "*" without a space between the variable.

For example:

char *ptr

instead of 

char * ptr

Thanks.